2021-04-26 - MIMUW submission

storaged opened the pull request:
Sorry, but my computation failed yesterday and I didn't notice.
Please, merge if possible.
The data used for the prediction are up to sunday (included)

sbfnk commented:
We're happy to merge this - can you remove the forecast from 19 April? We already have this via the submission to the German/Polish hub https://github.com/epiforecasts/covid19-forecast-hub-europe/pull/283

storaged commented:
Oh, I get it now. I was about to ask what was the problem with the last
week submission.
I will remove it asap

Thanks,
KGogolewski

wt., 27 kwi 2021, 15:12 użytkownik Sebastian Funk ***@***.***>
napisał:

> We're happy to merge this - can you remove the forecast from 19 April? We
> already have this via the submission to the German/Polish hub #283
> <https://github.com/epiforecasts/covid19-forecast-hub-europe/pull/283>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/epiforecasts/covid19-forecast-hub-europe/pull/314#issuecomment-827595765>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAJFQQDQSRV7FFU6KXOCBODTK2Z37ANCNFSM43U6D4NQ>
> .
>

kathsherratt commented:
Hi @storaged , please could you delete the conflicting file (`2021-04-19-MIMUW-StochSEIR.csv`) from this pull request? We can then merge this in. 
Thanks,
Kath

storaged commented:
I removed the file and added this week predicitons. 
I hope it can all be merged at once

storaged commented:
Hi Kate,
I am sorry, but I removed it wrongly last time.
Now I removed and added latest predictions
Best, Krzysiek

On Mon, 3 May 2021 at 12:37, Kath Sherratt ***@***.***> wrote:

> Hi @storaged <https://github.com/storaged> , please could you delete the
> conflicting file (2021-04-19-MIMUW-StochSEIR.csv) from this pull request?
> We can then merge this in.
> Thanks,
> Kath
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/epiforecasts/covid19-forecast-hub-europe/pull/314#issuecomment-831175795>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAJFQQHJJGNJPNFI5U7KOSLTLZ4GBANCNFSM43U6D4NQ>
> .
>


-- 
Pozdrawiam,
Krzysztof Gogolewski

kathsherratt commented:
That's great, thanks Krzysiek. We will merge alongside all this week's forecasts.
